plugins/LanguagesManager/LanguagesManager.php

```diff
@@ -73,7 +74,10 @@ public function addLanguagesManagerToOtherTopBar(&$str)
     {
         // piwik object & scripts aren't loaded in 'other' topbars
         $str .= "<script type='text/javascript'>if (!window.piwik) window.piwik={};</script>";
-        $str .= "<script type='text/javascript' src='plugins/CoreHome/angularjs/menudropdown/menudropdown.directive.js'></script>";
+
+        $coreHomeUmd = Development::isEnabled() ? 'CoreHome.umd.js' : 'CoreHome.umd.min.js';
+        $str .= "<script type='text/javascript' src='plugins/CoreHome/vue/dist/$coreHomeUmd'></script>";
```

Comment:
I guess this might be not needed anymore. If I have seen that correctly, the CoreUpdater and Installation Controller are both now adding the vue js files themselves, so they should be available already.

Reply:
They're adding the core Vue files (ie, vue.js + matomopolyfills), but not the CoreHome UMD. That still needs to be added... but maybe it's weird to have it here in LanguagesManager? Otherwise some future widget might have to do the same thing and it gets added twice. I'll just load it in CoreUpdater/Installation.